README.md

```diff
@@ -30,6 +30,10 @@ Supported identifiers include: device node (like `/dev/sda` or `/dev/mapper/mpat
 device node basename (like `sda` or `mpathb`), /dev/disk/ symlink
 (like `/dev/disk/by-id/wwn-0x5000c5005bc37f3f`).
 
+##### `raid_level`
+When used with `type: lvm` it manages a volume group with a mdraid array of given level
+on it. Accepted values are: `linear`, `striped`, `raid0`, `raid1`, `raid4`, `raid5`, `raid6`, `raid10`
+
```

Comment:
I am missing here a description on how to specify member drives of a RAID.
Normally, the `disks` are a list of disks to be used as PVs of the volume group. But in the case of RAID, they change their meaning: there will be a single PV and the `disks` determine the components of the RAID. This is not completely intuitive and warrants a better description.
Also, `raid10` is two-level - how do you specify which disks belong together in mirrors? https://en.wikipedia.org/wiki/Nested_RAID_levels#RAID_10_.28RAID_1.2B0.29

Reply:
Fixed: Added explicit note about `disks` behavior when `raid_level` option is used.

RAID handling is passed to blivet. At this point there is no additional logic regarding `raid10`.

Reply:
What does blivet do for raid10, then?

Reply:
The layout (which could perhaps use a brief explanation in README.md), is the usual arrangement: a single array is created across the specified disks and used as the lone member device for the pool.

The details of a RAID10 layout are handled by `mdadm(8)`. There is currently no way to override mdadm's default for that.